src/static/css/index.css

```diff
@@ -77,14 +55,14 @@ p {
   grid-template-areas: 'intro image';
   grid-gap: 2rem;
   justify-content: space-between;
-  padding: 0 60px;
 }
 
 .intro {
   -ms-grid-column: 1;
   grid-area: intro;
   width: 100%;
-  max-width: 420px;
+  max-width: 440px;
+  padding-left: 20px;
```

Comment:
Not sure why we need this, but need to remove it on mobile as causing everything to be off-center:

![Mobile view with padding](https://user-images.githubusercontent.com/10931297/89780755-f338dc80-db09-11ea-9dc8-6da48edbe541.png)

Reply:
Good catch ;) I've fixed that.

The left (padding) 20px  magic number is needed to match the .container side padding for the 900 - 1280px screen res scenario. For the right side padding, considering the .intro-image-wrapper image design, I've skipped it.

<img width="802" alt="sides-padding" src="https://user-images.githubusercontent.com/1867900/89802861-bcc78580-db3a-11ea-8434-9ae69797682f.png">

Reply:
Thanks. Agree it’s not really needed but is might be less confusing to put the right side in too. Might also stop that red guys butt going right to the edge. WDYT?

Reply:
IMO adding a right 20px `padding` would make things a bit worse here, as there's already too much extra space to the right, due to the current year image's design. 

900px - 1280px is an edge case after all, but if it were to be 100% correct, I think we'd need to add a margin-right: -40px or so to the `.intro-image-wrapper` to make things balanced here. The result here would be the red guy butt a bit cut, just as the below contributors number - which might look good I think :)